src/components/pages/Incidents/Timeline/timeline.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { Timeline, DatePicker, Space } from 'antd';
+import LocalPopOver from '../LocalPopOver';
+
+const { RangePicker } = DatePicker;
+
+const TimelineLabel = props => {
+  const [endDate, setEndDate] = useState('');
+  const [startDate, setStartDate] = useState('');
+  let mode = 'left';
+  const incidents = props.incidents;
+  const [filteredIncidents, setFilteredIncidents] = useState(incidents);
+  console.log(filteredIncidents);
+  if (window.screen.width >= 768) {
+    mode = 'alternate';
+  }
+
+  useEffect(() => {
+    if (startDate && endDate) {
+      const results = incidents.filter(el => {
+        const d = Date.parse(el.date);
+        const start = Date.parse(startDate);
+        const end = Date.parse(endDate);
+        console.log('Start', start, 'End', end);
+        return d >= start && d <= end;
+      });
+      setFilteredIncidents(results);
+    } else {
+      setFilteredIncidents(incidents);
+    }
+  }, [startDate, endDate, incidents]);
+
+  const makeTimeline = () => {
+    if (filteredIncidents.length > 0) {
+      return filteredIncidents.map(incident => {
+        return (
+          <Timeline.Item>
+            <LocalPopOver incident={incident} marker={false} text={true} />
+          </Timeline.Item>
+        );
+      });
+    }
+  };
+
+  const setStartValue = (value, dateString) => {
+    setStartDate(dateString);
+    console.log('value', dateString);
+  };
+
+  const setEndValue = (value, dateString) => {
+    setEndDate(dateString);
+    console.log('value', dateString);
+  };
+
+  const newPickDate = () => {
+    return (
+      <Space direction="vertical">
+        <DatePicker onChange={setStartValue} />
+        <DatePicker onChange={setEndValue} />
```

Comment:
I like this formatting a lot better, much easier to read without having the function written inline